lib/experimental/Widgets/Widget/index.tsx

```diff
@@ -60,91 +61,93 @@ const Container = forwardRef<
   }
 
   return (
-    <Card className="flex gap-4" ref={ref}>
-      {header && (
-        <CardHeader>
-          <div className="flex flex-1 flex-col gap-4 truncate">
-            <div className="flex flex-row items-center justify-between gap-2">
-              <div className="flex min-h-6 grow flex-row items-center gap-1.5 truncate">
-                {header.title && <CardTitle>{header.title}</CardTitle>}
-                {header.subtitle && (
-                  <>
-                    <InlineDot />
-                    <CardSubtitle>{header.subtitle}</CardSubtitle>
-                  </>
+    <PrivacyModeProvider>
+      <Card className="flex gap-4" ref={ref}>
```

Comment:
Notice how we've added this `PrivacyModelProvider` to our widget, so that we can always have this option to control it at widget level without having to use the provider at all

Reply:
I think we should just put this provider alongside the rest of the providers in the `FactorialOne` provider umbrella. This way when you set privacy = off in one of the widgets, it will happen on the rest of the app accordingly :).

I think this might be a bit weird at the moment since we have the "eye" button inside the widget, but this should be moved eventually to the layout itself /cc @dani-moreno .

Reply:
BTW this provider might benefit from a prop that sets an initial state like `initialPrivacy`, so we can then decide if we store it on `localStorage`, user settings, etc (because the app can inject that value when instantiating `<FactorialOne />`.